exporter/awsemfexporter/util.go

```diff
@@ -31,6 +31,7 @@ var patternKeyToAttributeMap = map[string]string{
 	"TaskId":               "aws.ecs.task.id",
 	"NodeName":             "k8s.node.name",
 	"PodName":              "pod",
+	"job":                  "job",
```

Comment:
```suggestion
	"ServiceName":                  "service.name",
```

I think I would be much more comfortable exposing `ServiceName`.  This is a required resource attribute in all OTel resources and the `prometheus` receiver will store the Prometheus job name in the `service.name` attribute of resources it emits.  I think this gets maybe a bit potentially confusing, though, as the existing options here mix ECS and k8s attributes and includes `NodeName` and `PodName` and "service" is a k8s concept.  Thoughts?

Reply:
I think `ServiceName` could work as well.
Agree it could be a little confusing as Service Name means something totally diff in the context of ECS but given we already have a mix of K8s and ECS specific tokens in here, a consumer of this functionality needs to be self-aware of what they are setting the label to be - so it might be alright?